app/views/layouts/decidim/_head_extra.html.erb

```diff
@@ -161,4 +161,11 @@
   }
 
 </style>
+
+<%# if Rails.env.production? %>
+  <script type='text/javascript'>
+    window.SENTRY_DSN = "<%= ENV["SENTRY_FRONTEND_DSN"] %>"
```

Comment:
I would replace SENTRY_DSN in .env.sample with SENTRY_FRONTEND_DSN

Reply:
Adjusted in 8a204cc.